### ISSUE TITLE
Support multiline docstring

### ISSUE THREAD

**Issue by ikitommi** [xadecimal/expose-api#1]
Importing `ring.util.response/status`

original:

```clojure
(defn file-response
  "Returns a Ring response to serve a static file, or nil if an appropriate
  file does not exist.
  Options:
    :root            - take the filepath relative to this root path
    :index-files?    - look for index.* files in directories (defaults to true)
    :allow-symlinks? - allow symlinks that lead to paths outside the root path
                       (defaults to false)"
  ([filepath]
   (file-response filepath {}))
  ([filepath options]
   (if-let [file (find-file filepath options)]
     (let [data (file-data file)]
       (-> (response (:content data))
           (content-length (:content-length data))
           (last-modified (:last-modified data)))))))
```

imported:

```clojure
(defn file-response
  "Returns a Ring response to serve a static file, or nil if an appropriate\n  file does not exist.\n  Options:\n    :root            - take the filepath relative to this root path\n    :index-files?    - look for index.* files in directories (defaults to true)\n    :allow-symlinks? - allow symlinks that lead to paths outside the root path\n                       (defaults to false)"
  {:inline (fn
             ([filepath] ` (ring.util.response/file-response ~filepath))
             ([filepath options]
              `
              (ring.util.response/file-response ~filepath ~options)))}
  ([filepath] (ring.util.response/file-response filepath))
  ([filepath options] (ring.util.response/file-response filepath options)))
```

**Comment by didibus** [comment]
I think this might be an issue with zprint, I'll see if I can workaround it, but also raised an issue there: https://github.com/kkinnear/zprint/issues/326

While in the file it looks a bit ugly to the eye, at least in Emacs with lsp, and also cider, both when they show the doc it is properly multiline, because they parse back the \n and display it as a real newline.

**Comment by didibus** [comment]
Fixed in 0.2.0
